userspace/plugin/plugin_types.h

```diff
@@ -31,8 +31,15 @@ typedef uint32_t ss_plugin_bool;
 // falcosecurity libs types.
 typedef enum ss_plugin_field_type
 {
-	FTYPE_UINT64 = 8,
-	FTYPE_STRING = 9
+	FTYPE_UINT64      = 8,
+	FTYPE_STRING      = 9,
+	FTYPE_IPV4ADDR    = 12,
```

Comment:
what is the reason behind these numbers? at first glance, it seems they are matching the `ppp_param_type` enum but `PT_IPV4ADDR` is 26 while here is 12 :thinking:

Reply:
Ops! My bad. I'll fix it also in the sdk. The comment it's already there:
https://github.com/falcosecurity/libs/blob/1299cb4de1515d2aa5dc8d636787be728f127779/userspace/plugin/plugin_types.h#LL30C1-L30C1